content/influxdb/cloud-dedicated/get-started/write.md

```diff
@@ -328,17 +409,17 @@ Include the following with your request:
 - **Request body**: Line protocol as plain text
 
 {{% note %}}
-With the {{% product-name %}} v2 API `/api/v2/write` endpoint, `Authorization: Bearer` and `Authorization: Token` are equivalent and you can use either scheme to pass a database token in your request. For more information about HTTP API token schemes, see how to [authenticate API requests](/influxdb/cloud-dedicated/guides/api-compatibility/v2/).
+With the {{% product-name %}} v2 API `/api/v2/write` endpoint, `Authorization: Bearer` and `Authorization: Token` are equivalent and you can use either scheme to pass a database token in your request.
```

Comment:
I don't know the technical differences between the `Bearer` and `Token` schemes, but I hesitate to use "equivalent" here.

```suggestion
The {{% product-name %}} v2 API `/api/v2/write` endpoint supports both `Bearer` and `Token` authorization schemes and you can use either scheme to pass a database token in your request.
```